layouts/community/ergodox/replicaJunction/config.h

```diff
@@ -1,64 +1,19 @@
-/*
-Config file - Ergodox QMK with replicaJunction layout
-
-This program is free software: you can redistribute it and/or modify
-it under the terms of the GNU General Public License as published by
-the Free Software Foundation, either version 2 of the License, or
-(at your option) any later version.
-
-This program is distributed in the hope that it will be useful,
-but WITHOUT ANY WARRANTY; without even the implied warranty of
-MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-GNU General Public License for more details.
-
-You should have received a copy of the GNU General Public License
-along with this program.  If not, see <http://www.gnu.org/licenses/>.
-*/
-
-#ifndef KEYBOARDS_ERGODOX_CONFIG_H_
-#define KEYBOARDS_ERGODOX_CONFIG_H_
+#ifndef CONFIG_USER_H
```

Comment:
Could you replace the first few lines, eg, these lines: 
```c
#ifndef CONFIG_USER_H
#define CONFIG_USER_H
```
with just `#pragma one`, and remove the endif at the end?